viz-lib/src/visualizations/chart/plotly/prepareHeatmapData.js

```diff
@@ -22,6 +22,10 @@ function prepareSeries(series, options, additionalOptions) {
     type: "heatmap",
     name: "",
     colorscale: colorScheme,
+    xgap: options.heatmapSpacing,
```

Comment:
UX-wise, I came to find this unit (probably pixel) too sensitive, as decimal values provide a more reasonable range than integers. Maybe consider adding a constant ratio for that.

Reply:
How do you like the sensitivity now in 916374e?